travis/small-requirements.txt

```diff
@@ -1,22 +1,16 @@
 # Small test reqs
-azure-storage-blob==12.3.0
 google-cloud-storage==1.14.0
-botocore==1.12.84  # pinned for moto
-boto3==1.9.84      # pinned for moto
-mock==2.0.0
+botocore==1.12.84
+boto3==1.9.84
 moto==1.3.7
-mxnet==1.5.0
 pandas<=0.23.4
```

Comment:
Couldn't remove pandas here due to this issue https://github.com/mlflow/mlflow/issues/3006.